apps/examples/wifi_manager/wm_test/wm_test_table.h

```diff
@@ -25,3 +25,4 @@ WT_MEMBER_POOL(WT_TEST_ADD_AP, _wt_interop_add_ap, _wt_parse_set, "add_ap")
 WT_MEMBER_POOL(WT_TEST_DISPLAY_AP, _wt_interop_display_ap, _wt_parse_interop, "display_ap")
 WT_MEMBER_POOL(WT_TYPE_EVT_TC, _wt_run_evt_tc, _wt_parse_evt_tc, "tc_evt")
 WT_MEMBER_POOL(WT_TYPE_DNS, _wt_dns_test, _wt_parse_dns, "dns")
+WT_MEMBER_POOL(WT_TYPE_CONNECTBYRSSI, _wt_connectbyrssi_test, _wt_parse_connectbyrssi, "connect_by_rssi")
```

Comment:
please check `no newline at end of file` warning.

Reply:
Done